src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.props

```diff
@@ -14,11 +14,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!-- Default properties that shouldn't be replaced by the microsoft.net.sdk.props -->
   <PropertyGroup>
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
-    <PreserveCompilationContext Condition="'$(PreserveCompilationContext)' == ''">true</PreserveCompilationContext>
```

Comment:
@rynowak - this is from the meeting with Damian. Here is what I think this would look like, correct me if I missed how it would go:

* Move this property to the RazorSdk
* This value is set in a target if you're `OutputType = executable` and there are any cshtml files in the project

Reply:
Yup.